autocompleter/base.py

```diff
@@ -476,6 +477,11 @@ def suggest(self, term):
             for provider in providers:
                 total_results += registry.get_ac_provider_setting(self.name, provider, 'MAX_RESULTS')
 
+        # Generate a unique identifier to be used for storing intermediate results. This is to
+        # prevent redis key collisions between competing suggest / exact_suggest calls.
+        uuid_str = str(uuid.uuid4())
+        base_intermediate_result_set = 'djac.results.%s' % (uuid_str,)
```

Comment:
I would define 'djac.results.%s' as RESULT_SET_BASE_NAME above like we do with other similar constants...
So then it would be like
result_key = RESULT_SET_BASE_NAME % (uuid_stir,)

and then below
instead of:
```
result_key = "djac.results.%s.%s" % (uuid_str, norm_term,)
```
```
term_result_key = result_key + '.' + norm_term
```